src/components/ItemList/ItemList.module.css

```diff
@@ -0,0 +1,12 @@
+.wrap {
+  padding-bottom: 15px;
+}
+
+.item__wrap { 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5693406)